index.html

```diff
@@ -187,20 +187,89 @@ <h2>Name and Description</h2>
-    <p><a class="termref">User agents</a> MUST compute an <a class="termref">accessible name</a> using the rules outlined below in the section titled <a href="#mapping_additional_nd_te">Accessible Name and Description Computation</a>.</p>
+    <p><a class="termref">User agents</a> MUST compute an <a class="termref">accessible name</a> using the rules outlined below in the section titled <a href="#mapping_additional_nd_te">Text Equivalent Computation</a>.</p>
   </section>
+
   <section id="mapping_additional_nd_description">
     <h3>Description Computation</h3>
-    <p>If <a class="property-reference" href="#aria-describedby"><code>aria-describedby</code></a> is present, <a class="termref">user agents</a> MUST compute the accessible description by concatenating the text alternatives for elements referenced by an <code>aria-describedby</code> attribute on the current element. The text alternatives for the referenced elements are computed using a number of methods, outlined below in the section titled <a href="#mapping_additional_nd_te">Accessible Name and Description Computation</a>. </p>
+    <p>The following table provides the order of precedence for markup that can be applied to compute an <a class="termref">accessible description</a>. <a class="termref">User agents</a> MUST use the first applicable entry from the table where the listed conditions are met, as described in the last column. The user agent MUST NOT use any markup other that the first relevant markup found, even if that markup results in an empty description:</p>
+    <table>
+      <thead>
+        <tr>
+          <th>Precedence</th>
+          <th>Attribute</th>
+          <th>Applicable conditions</th>
+          <th>How used to compute description</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>1</th>
+          <td><a class="property-reference" href="#aria-describedby"><code>aria-describedby</code></a> attribute</td>
+          <td>Use on any element</td>
+          <td><a href="#mapping_additional_nd_te">Text content computation</a> on all nodes referenced by aria-describedby on the element, concatenated, and separated by a space character</td>
+        </tr>
+        <tr>
+          <th>2</th>
+          <td><a class="property-reference" href="#aria-describedby"><code>aria-description</code></a> attribute</td>
+          <td>Use on any element</td>
+          <td>As a flat string</td>
+        </tr>
+        <tr>
+          <th>3</th>
+          <td>HTML <code>value</code> attribute</td>
+          <td>
+            <ul>
+              <li>Only use on HTML <code>&lt;input type="button"|"submit"|"reset"&gt;</code></li>
+              <li>Only use if not already used for the accessible name of this node</li>
+            </ul>
+          </td>
+          <td>As a flat string</td>
+        </tr>
+        <tr>
+          <th>4</th>
+          <td>HTML <code>&lt;caption&gt;</code> child of a <code>&lt;table&gt;</code></td>
```

Comment:
Isn't `caption` already covered as a ~"host language labeling mechanism" for `table`?

Regardless, I think it's okay to use `html:caption` as the only known example instance of a more specific rule, but the rule should not limit itself to HTML.

Reply:
```suggestion
          <td>HTML <code>&lt;caption&gt;</code> child of a <code>&lt;table&gt;</code> or equivalent host lang feature</td>
```